app/views/directives/add-config-to-application.html

```diff
@@ -108,7 +111,7 @@
                   class="btn btn-primary"
                   ng-class="{'dialog-btn': isDialog}"
                   ng-click="ctrl.addToApplication()"
-                  ng-disabled="ctrl.addType === 'volume' && addToApplicationForm.$invalid || !ctrl.application"
+                  ng-disabled="(ctrl.addType === 'volume' && addToApplicationForm.$invalid) || (ctrl.addType === 'env' && !ctrl.canAddRefToApplication)"
```

Comment:
I think you still need to ensure an application is selected either way.

Reply:
@jeff-phillips-18 I've update the mount path input with the check if the application was selected as @spadgett suggested. This will force user to pick the application in order to make the form valid. Added separate commit so the changes are obvious.
PTAL